### ISSUE TITLE
Error on deploying last commit from v.22.05

### ISSUE THREAD

**Issue by juanmimjimenez** [galaxyproject/galaxy#15839]
Hello,
I have some ansible playbooks that were working all right.
I have run it in a new VM and with commit #5dc2c4f the gunicorn server does not start, this didn't happen before. I used this script two weeks ago on a different VM.

Galaxy Version: 22.05
Commit: 5dc2c4f8c32656c25265f8ede086863ad11c

On gunicorn log I get this message:
  File "/srv/galaxy/server/lib/galaxy/model/migrations/__init__.py", line 304, in verify_databases
    gxy_verifier.run()
  File "/srv/galaxy/server/lib/galaxy/model/migrations/__init__.py", line 360, in run
    self._handle_nonempty_database()
  File "/srv/galaxy/server/lib/galaxy/model/migrations/__init__.py", line 385, in _handle_nonempty_database
    self._handle_no_version_table()
  File "/srv/galaxy/server/lib/galaxy/model/migrations/__init__.py", line 474, in _handle_no_version_table
    raise NoVersionTableError(model)
galaxy.model.migrations.NoVersionTableError: Your galaxy database has no version table; manual update is required

The VM didn't have a running database. It was an empty server.

Any hints?
Thanks

**Comment by martenson** [comment]
Hi @juanmimjimenez it is a bit unclear to me what the issue is. However it does not seem related to the commit you linked. Can you roll back to the commit you had before when everything worked fine? What commit was that?

As a side note, even if you do not set up database like postgres Galaxy is still going to create an empty file-based sqlite database so the error message is most likely relevant.

**Comment by juanmimjimenez** [comment]
I will check on another environment the commit that was working. When I deploy the ansible I setup the tag of the release like release_22.05, will it work the same if I setup the specific commit hash?
Thanks

**Comment by juanmimjimenez** [comment]
Hi again:
I am no expert but it seems that there is an error between the commit 054d2c71a2bba7ac7d56fbc1fcd98c9f8f00b85e  (march 14) and the commit of march 20, 5dc2c4f8c32656c25265f8ede086863ad11c5844.
With the commit of the 20 march I get the above error about the database has no version table, I have checked twice, making snapshots on VM and rolling back.
Then I forced the installation of the commit 054d2c71a2bba7ac7d56fbc1fcd98c9f8f00b85e from march 14 and galaxy installed successfully, and the only change in the process was that. Then from that version I updated to the march 20 one and it updated with no problem.
